src/main.js

```
@@ -30,55 +31,128 @@ scene.add(new THREE.AmbientLight(0x666666));
+    }
+    // --- NEW: Control lockout ---
+    if (aircraft.isOnFire || aircraft.crashed) {
+      aircraft.controlsLocked = true;
+    } else {
+      aircraft.controlsLocked = false;
+    }
+  }
 
-    if (missionManager) missionManager.update();
+  // --- NEW: Timer triggers ---
+  if (aircraft && aircraft.airborne && !window.timerStarted) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)